operator/cmd/mesh/testdata/manifest-generate/output/pilot_default.golden.yaml

```diff
@@ -1849,14 +1849,15 @@ data:
                   successThreshold: 1
                   timeoutSeconds: 1
                 securityContext:
-                  privileged: true
+                  privileged: false
```

Comment:
Is this related to "drop legacy telemetry install"? 
Same question for other changes under operator/cmd/mesh/testdata/manifest-generate/output.

Reply:
No -- this is from 'Update operator "snapshots" tests, which required a few seemingly unrelated changes to tests.'

Basically, the tests here are based on a _snapshot_ of the manifests/ folder. This snapshot needs to be updated in this PR due to the size of the changes, but is not routinely update. So its pulling in changes from the past few weeks (at one point there was none, but this PR has been open for a while)